### ISSUE TITLE
不同batch size时inference结果不同

### ISSUE THREAD

**Issue by Wunaiq** [woshidandan/TANet#8]
#3 中提到的问题1会导致分类的支路输出的结果与batch size有关，当batch size是不同时，inference结果也不同，并且bs=1时，分类的支路输出固定为1；

请问怎么解决不同batch size时inference结果不同的问题呢？
在端侧使用的时应该bs=1的情况很多，但此时分类的分支输出是固定为1的，就起不到作用了吧？

**Comment by woshidandan** [comment]
您好！感谢您对这项工作的持续关注，刚才我试了一下用不同的batch size，在TAD66K上测试了推理的性能，结果如下：
    # test_loader = DataLoader(test_ds, batch_size=64, num_workers=opt['num_workers'], shuffle=False)
    # lcc_mean: 0.5266745930806195, srcc_mean: 0.5069025614121254, validate_losses: 0.015697210137563315
    
    # test_loader = DataLoader(test_ds, batch_size=32, num_workers=opt['num_workers'], shuffle=False)
    # lcc_mean: 0.5264128589068133, srcc_mean: 0.5066027710957562, validate_losses: 0.015706565844425466
    
    test_loader = DataLoader(test_ds, batch_size=1, num_workers=opt['num_workers'], shuffle=False)
    # lcc_mean: 0.5258770707185415, srcc_mean: 0.5068998155002807, validate_losses: 0.01672296673651432
    
    inference的结果几乎没有差别，细微的差别可能来自于推理设备不同和网络中相关随机层参数没有被固定的原因。当时给公司做开发，所以用来推理的batch size等一些参数，都是甲方根据需求来定，一般batch size不会是1。
